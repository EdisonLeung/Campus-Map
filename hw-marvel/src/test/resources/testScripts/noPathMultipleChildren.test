# Test pathfinding for a node where there is no path to another node but both nodes
# have children (they are just not related)

# Create graph
CreateGraph noPath

# Add nodes
AddNode noPath a
AddNode noPath b
AddNode noPath c

# Add edges
AddEdge noPath a b ABa
AddEdge noPath a b ABb
AddEdge noPath c a CA
AddEdge noPath c b CB

# pathfinding from a to c
FindPath noPath a c