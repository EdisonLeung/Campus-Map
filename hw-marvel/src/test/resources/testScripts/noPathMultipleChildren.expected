# Test pathfinding for a node where there is no path to another node but both nodes
# have children (they are just not related)

# Create graph
created graph noPath

# Add nodes
added node a to noPath
added node b to noPath
added node c to noPath

# Add edges
added edge ABa from a to b in noPath
added edge ABb from a to b in noPath
added edge CA from c to a in noPath
added edge CB from c to b in noPath

# pathfinding from a to c
path from a to c:
no path found